src/domain/models/Message.ts

```diff
@@ -36,10 +36,10 @@ export class Message implements Pluto.Storable {
       1 * 24 * 60 * 60
     ).toString(),
     public readonly ack: string[] = [],
-    public readonly direction: MessageDirection = MessageDirection.RECEIVED,
+    public direction: MessageDirection = MessageDirection.RECEIVED,
```

Comment:
why is this no longer readonly?

Reply:
Because instead of requiring the user's to set MessageDirection.SENT manually on messages this is done automatically by the SDK int he ConnectionsManager.ts, sendMessage fn.

This was not being set correctly so all the messages are received always, this fixes it.